### DISCUSSION TITLE
Showing Reverse Relationships for Elements in the CP

### DISCUSSION THREAD

**Discussion by BrandKits** [craftcms/cms#8850] [Ideas]
Craft has a powerful engine for relating elements to one another... on the front end. However, content authors can't see what is related in the CP.

The use case that prompted this was showing Products < > Categories. The client wanted to see/add products to a category, and then on the product, see what categories have been applied.

But we all have uses cases, to show:

Entries < > Products
Entries < > Users
And more.

As a feature request, perhaps there could be an option when setting up a field to show all element relations. 

Maybe with a table view with title, element type, CP link (perhaps customisable columns).

This way content authors can clearly see how their content is related in the CP. 

Powerful relations, front and back ends.

**Comment by brandonkelly** [comment]
Kindof related to #3907, but I’ll leave this open as all you’re asking for it to _see_ the relations, which would certainly be easier to implement than actually _managing_ the relations from the other end.

In the meantime, check out the [Relations](https://plugins.craftcms.com/relations) plugin, which provides a field type that displays reverse relations.

**Comment by BrandKits** [comment]
You have a good point, the client would really want to view/add/delete the related elements, from both sides. Overall I think it would be very intuitive for content authors (who never know where to go in the CMS to do something) and add to their love of Craft.

**Comment by daltonrooney** [comment]
You don't even need a plugin to _display_ related entries any more. You can make use of the new UI Elements feature in Craft to create a list of related elements in the edit panel: http://craftsnippets.com/articles/using-template-ui-elements-to-extend-craft-cms-control-panel#displaying-reverse-element-relations

I agree that it would be useful to _manage_ reverse relations, but that's a different issue.

**Comment by lassemt** [comment]
I like the above solution for showing the relations, but it would be nice to have something like ["Reverse Relations plugin for Craft"](https://github.com/robuust/craft-reverserelations) in core when querying entries as well.
